validator/weight_setting/validator.py

```diff
@@ -671,13 +668,13 @@ def connect_to_api(self):
 
         return websocket
 
-    def api_logs(self):
+    async def api_logs(self):
         while True:
             try:
                 for line in self.websocket:
                     output = sys.stderr if line.startswith("err:") else sys.stdout
 
-                    print(line[4:], file=output)
+                    print(line, file=output)
```

Comment:
The 4 character prefix is the aforementioned `out:` and `err:` which should probably be kept